templates/impala/search.html

```diff
@@ -16,7 +19,10 @@
     {{ search_form.platform }}
   {% endif %}
   <button class="search-button" type="submit" title="{{ _('Search') }}"
-         src="{{ static('img/zamboni/global/btn-search.png') }}"></button>
+         src="{{ static('img/zamboni/global/btn-search.png') }}"
+          style="text-indent:-999em;">search
```

Comment:
I am completely lost in css files, most of them are read-only and I can't find a class that hides text, so I need to create a new one, which css file should I define the new class at ?